tests/integration/memcached/test_socket.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should report instance metrics when multiple servers are
+added at once
```

Comment:
This is not accurate.

Reply:
fixed in [3364eb0](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/3364eb0b8bbf0e984add9030588f82ff3475ed05)